modules/stanford_news_extras/stanford_news_extras.install

```diff
@@ -0,0 +1,42 @@
+<?php
+
+
+/**
+ * Implements hook_install().
+ */
+function stanford_news_extras_install() {
+
+  // Get the default banner image
+  $filename = "news-extra-default-image.png";
+  $file_path = drupal_get_path('module', 'stanford_news_extras') . "/images/" . $filename ;
+
+  $image = file_get_contents($file_path);
+
+  //Creates a database entry.
+  $file = file_save_data($image, "public://" . $filename, FILE_EXISTS_REPLACE);
+
+  //Set the file status to permanent so it is not deleted in next cron run
+  $file->status = FILE_STATUS_PERMANENT;
```

Comment:
This additional setting and save is not needed. 

The default value for this is FILE_STATUS_PERMANENT. Please see:
https://api.drupal.org/api/drupal/includes%21file.inc/function/file_save_data/7.x